### PR TITLE
hit my time limit got some of the code started

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/tree-intersection/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+working ll
+working tree traversal
+test to make sure the funtion take in trees right
+make sure the trees are added to the hashmap
+make sure the hash map deal with collisions
+test that the function looks for duplicates in trees as it is inserting in to the map and removing them.
+
+## Solution
+
+![whiteBoard]('./assets/CC-whiteboard.jpg');
```
Comment (adrianhuebner) [inline javascript/code-challenges/tree-intersection/README.md:26]: Everything is looking good, would just like to see tests added
